configs/cee/packages.yaml

```
@@ -9,8 +9,6 @@ packages:
     version: [1.80.0]
   cmake:
     externals:
-    - spec: cmake@3.22.2
-      prefix: /projects/wind/system-spack/opt/spack/linux-rhel7-x86_64/gcc-9.3.0/cmake-3.22.2-s67tgoaqaw4ky5kmpuzxdg5une7xouvh/
     - spec: cmake@3.25.0%clang@13.0.1~doc+ncurses+ownlibs~qt build_type=Release arch=linux-rhel7-x86_64
```

I think this issue is the arch in these external specs. The ubuntu runner is not rhel7-x86-64 so it is likely failing b/c the specs are too restrictive
```suggestion
    - spec: cmake@3.25.0%clang@13.0.1~doc+ncurses+ownlibs~qt build_type=Release
```